modules/govuk/manifests/apps/ckan.pp

```diff
@@ -173,5 +184,17 @@
       owner  => 'deploy',
       group  => 'deploy',
     }
+
+    exec { 'symlink_pycsw_config':
+      command => "test -e ${ckan_home}/pycsw.cfg || ln -s ${pycsw_src}/default.cfg ${ckan_home}/pycsw.cfg",
+    }
```

Comment:
How come the config is symlinked, rather than just put in `${ckan_home}/pycsw.cfg` in the first place?

Reply:
The pycsw config is placed there and symlinked in order to not confuse people who attempt to update it in `/var/ckan` and also to keep configs in a common area, alternatively we could set an env var `PYCSW_CONFIG` to point to the config area but wasn't sure how to do this.

Reply:
Looks like a symlink isn't being used now.